db/migrate/20231002025529_add_fields_to_users.rb

```diff
@@ -0,0 +1,20 @@
+class AddFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :unsubscribed_at, :datetime
+    add_column :users, :unsubscribed_reason, :string
+    add_column :users, :available_as_mentor_at, :datetime
+    add_column :users, :requested_mentorship_at, :datetime
+    add_column :users, :city, :string
+    add_column :users, :country_code, :string
+    add_column :users, :lat, :decimal, precision: 10, scale: 6
+    add_column :users, :lng, :decimal, precision: 10, scale: 6
+    add_column :users, :demographic_year_of_birth, :integer
+    add_column :users, :demographic_year_started_ruby, :integer
+    add_column :users, :demographic_year_started_programming, :integer
+    add_column :users, :demographic_underrepresented_group, :boolean
```

Comment:
I opted not to implement any null: false constraints at the migration/database level because the User is created before we route to the onboarding mentor/mentee questionnaire. As a result, initially, these fields will be empty.

Reply:
Works for me.